desktop/src/main/java/bisq/desktop/main/dao/wallet/BsqBalanceUtil.java

```diff
@@ -191,4 +212,28 @@ public int addBondBalanceGroup(GridPane gridPane, int gridRow, String groupStyle
 
         return gridRow;
     }
+    
+    private void sumReputationBsq() {
+		Context ctx = Context.get();
+		new Thread(() -> {
+			Context.propagate(ctx);
+	        List<Transaction> walletTransactions = bsqWalletService.getClonedWalletTransactions();
+	        long reputationSum = walletTransactions.stream().map(transaction -> {
+	                    return new BsqTxListItem(transaction,
+	                            bsqWalletService,
+	                            btcWalletService,
+	                            daoFacade,
+	                            // Use tx.getIncludedInBestChainAt() when available, otherwise use tx.getUpdateTime()
+	                            transaction.getIncludedInBestChainAt() != null ? transaction.getIncludedInBestChainAt() : transaction.getUpdateTime(),
+	                            bsqFormatter);
+	                }).filter(new Predicate<BsqTxListItem>() {
+	        			@Override
+	        			public boolean test(BsqTxListItem t) {
+	        				return TxType.COMPENSATION_REQUEST.equals(t.getTxType()) || TxType.REIMBURSEMENT_REQUEST.equals(t.getTxType());
+	        			}
+	        		}).mapToLong(i -> i.getAmount().getValue()).sum();
+	        
+	        reputationBalanceTextField.setText(bsqFormatter.formatCoinWithCode(Coin.valueOf(reputationSum)));
+		}).start();
+    }
```

Comment:
If you refer with reputation to the merit used in voting this is not a correct implementation.
The merit has a decline function over 2 years. The DAO domain contains methods to get that correct value (see in UI code for voting as there it is displayed).
Also you should avoid to call the UI thread from other threads, can cause issues.